ha/cli/exec/commandExecutor.py

```diff
@@ -22,11 +22,25 @@
 
 from ha import const
 from cortx.utils.log import Log
+from cortx.utils.conf_store.conf_store import Conf
 from ha.core.error import HAInvalidPermission
+from ha.core.config.config_manager import ConfigManager
+from ha.core.controllers.element_controller_factory import ElementControllerFactory
+
 
 class CommandExecutor:
     def __init__(self):
         self._is_hauser = False
+        self._cluster_type = Conf.get(const.HA_GLOBAL_INDEX, "CLUSTER_MANAGER.cluster_type")
+        self._env = Conf.get(const.HA_GLOBAL_INDEX, "CLUSTER_MANAGER.env")
+        ConfigManager.load_controller_schema()
+        self._controllers = ElementControllerFactory.init_controller(self._env, self._cluster_type)
+        for controller in self._controllers.keys():
+            Log.info(f"Adding {controller} property to cluster manager.")
+            # Add property method for controller
+            # Example: cm.cluster_controller.start()
+            # Find more example in test case.
+            self.__dict__[controller] = self._controllers[controller]
```

Comment:
env and cluster type already handled in cluster manager

this not needed
only 
cm.cluster_controller.start()
should work

Reply:
Yes did same :) thank you..